src/routes/user/_user.transactions.tsx

```diff
@@ -0,0 +1,179 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { createFileRoute, useSearch } from '@tanstack/react-router';
+
+import { CircleX, Search } from 'lucide-react';
+
+import React, { useEffect, useState } from 'react';
+
+import { toast } from 'sonner';
+// import { RecordTxDialogAdmin } from 'src/components/RecordTxDialogAdmin';
+import { useDebounce } from 'use-debounce';
+
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  CardTitle,
+  DataTable,
+  Input,
+  PrepareTxDialogUser,
+  Tabs,
+  TabsContent,
+  TabsList,
+  TabsTrigger,
+  Toaster,
+  TransactionEditDialog,
+} from '@/components';
+import { columns } from '@/components/TransactionsColumns/columns.tsx';
+import { useSpvWalletClient } from '@/contexts';
+import { transactionsSearchSchema } from '@/routes/admin/_admin.transactions.tsx';
+import { transactionsUserQueryOptions } from '@/utils/transactionsUserQueryOptions.tsx';
+
+export const Route = createFileRoute('/user/_user/transactions')({
+  component: Transactions,
+  validateSearch: transactionsSearchSchema,
+  loaderDeps: ({ search: { order_by_field, sort_direction, blockHeight, createdRange, updatedRange } }) => ({
+    order_by_field,
+    sort_direction,
+    blockHeight,
+    createdRange,
+    updatedRange,
+  }),
+  loader: async ({
+    context: { queryClient, spvWallet },
+    deps: { sort_direction, order_by_field, blockHeight, createdRange, updatedRange },
+  }) =>
+    //TODO: add getDraftTransactions request
+    await queryClient.ensureQueryData(
+      transactionsUserQueryOptions({
+        spvWalletClient: spvWallet.spvWalletClient!,
+        sort_direction,
+        order_by_field,
+        blockHeight,
+        createdRange,
+        updatedRange,
+      }),
+    ),
+});
+
+function Transactions() {
+  const [tab, setTab] = useState<string>('all');
+  const [blockHeight, setBlockHeight] = useState<string>('');
+  const [debouncedBlockHeight] = useDebounce(blockHeight, 200);
+
+  const { spvWalletClient } = useSpvWalletClient();
+  const { order_by_field, sort_direction } = useSearch({ from: '/user/_user/transactions' });
+
+  useEffect(() => {
+    (async () => {
+      const res = await spvWalletClient?.GetTransactions({}, {}, {});
+      console.log('1', res);
```

Comment:
I think we should not add merge console.logs to adminV2. What if we forget about it and we merge `adminV2` to the main branch or we will treat `adminV2` as main branch.

Reply:
It has been removed